routes/userRoutes.js

```diff
@@ -0,0 +1,18 @@
+import express from 'express';
+import {
+  createUser,
+  getAllUsers,
+  getUserById,
+  updateUser,
+  deleteUser,
+} from '../controllers/userController.js';
+
+const router = express.Router();
+
+router.post('/users', createUser);
+router.get('/users', getAllUsers);
+router.get('/users/:id', getUserById);
+router.put('/users/:id', updateUser);
+router.delete('/users/:id', deleteUser);
```

Comment:
How about?
```suggestion
router.post('/', createUser);
router.get('/', getAllUsers);
router.get('/:id', getUserById);
router.put('/:id', updateUser);
router.delete('/:id', deleteUser);
```
and on [`app.js`](https://github.com/ArionDev8/house-b/pull/5/files#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28f)
```
app.use(express.json());
app.use('/users', userRoutes);
```